lib/web_ui/test/engine/pointer_binding_test.dart

```diff
@@ -2571,10 +2610,35 @@ mixin _MultiPointerEventMixin on _BasicEventContext {
   }
 }
 
+mixin _KeyboardModifiersMixin on _BasicEventContext {
```

Comment:
My memory to this file is getting vague, but since this mixin is used for all three event classes, can we just add these new properties to `_BasicEventContext`?

Reply:
> can we just add these new properties to `_BasicEventContext`?

Done, my first implementation was more complex so I did not want to clutter _BasicEventContent.